locales.json

```diff
@@ -24,244 +13,290 @@
       },
       "metaDescription": "Integreat ist Ihr digitaler Guide für Deutschland. Finden Sie lokale Informationen, Events und Beratung. Immer aktuell und in Ihrer Sprache."
     },
+    "ar": {
+      "pageTitles": {
+        "search": "البحث",
+        "events": "الفعاليات",
+        "extras": "العروض",
+        "landing": "اختيار المكان",
+        "disclaimer": "هيئة التحرير وجهة الاتصال",
+        "mainDisclaimer": "هيئة التحرير وجهة الاتصال لتطبيق إنتجريت",
+        "pois": "النقاط المثيرة للاهتمام",
+        "notFound": "تعذر العثور على الصفحة"
+      },
+      "metaDescription": "تطبيق إنتجريت هو دليلك الرقمي لألمانيا. تجد فيه معلومات وفعاليات محلية ويقدم لك المشورة. دائمًا مُحدَّثة وبلغتك."
+    },
     "en": {
       "pageTitles": {
         "search": "Search",
         "events": "Events",
         "extras": "Offers",
         "landing": "Location selection",
-        "disclaimer": "Imprint and contact",
-        "mainDisclaimer": "Imprint and Contact - Integreat",
-        "pois": "Points of Interest",
-        "notFound": "Seite konnte nicht gefunden werden."
+        "disclaimer": "Imprint and contacts",
+        "mainDisclaimer": "Imprint and contacts - Integreat",
+        "pois": "Points of interest",
+        "notFound": "Page not located"
```

Comment:
This does not seem right

Reply:
https://issues.integreat-app.de/browse/NATIVE-413